src/app/shared/model/feeder-episode.model.ts

```diff
@@ -52,6 +53,9 @@ export class FeederEpisodeModel extends BaseModel {
     this.authorName = author['name'] || '';
     this.authorEmail = author['email'] || '';
     this.summary = this.doc['summary'] || '';
+    if (this.doc['_links'] && this.doc['_links']['enclosure'] && this.doc['_links']['enclosure']['href']) {
+      this.enclosureUrl = this.doc['_links']['enclosure']['href'] || '';
```

Comment:
Should enclosure be a related model? I know it's right there in _links, but we don't access _links, we follow them.

Reply:
that is usually true. in this case the (content) type of the link is not to another json-hal resource, it's to the enclosure audio/mpeg. I see about 50/50 folks using _links for non-hal resources in their APIs, there is a `type` attr on the link object to support this way of using it.